frontend/app/src/components/history/ledger-actions/LedgerActionForm.vue

```diff
@@ -9,8 +9,6 @@ import {
   type NewLedgerAction
 } from '@/types/history/ledger-action/ledger-actions';
 import { LedgerActionType } from '@/types/history/ledger-action/ledger-actions-type';
-import { Zero, bigNumberifyFromRef } from '@/utils/bignumbers';
-import { convertFromTimestamp, convertToTimestamp } from '@/utils/date';
 import { toMessages } from '@/utils/validation-errors';
```

Comment:
is there any reason that `toMessages` can't be removed ?

Reply:
used inside `<template>`, so component will break since auto-import is mostly supported inside script

Reply:
this is only on vue 2, for vue 3 it works